libraries/ESP8266HTTPClient/src/ESP8266HTTPClient.cpp

```diff
@@ -21,15 +21,19 @@
  * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
  *
  */
-
 #include <Arduino.h>
+
+#include "ESP8266HTTPClient.h"
+
+#ifdef KEEP_PRESENT_API
 #include <ESP8266WiFi.h>
 #include <WiFiClientSecure.h>
+#endif
+
 #include <StreamString.h>
 #include <base64.h>
 
-#include "ESP8266HTTPClient.h"
-
+#ifdef KEEP_PRESENT_API
```

Comment:
Suggest making `KEEP_PRESENT_API` less ambiguous, for example `HTTPCLIENT_1_1_COMPATIBLE` (referring to current 1.1 version of the library).

Reply:
Chaning KEEP_PRESENT_API into HTTPCLIENT_1_1_COMPATIBLE